lib/gateway.js

```diff
@@ -446,7 +460,7 @@ async function buildGateway (gatewayOpts, app) {
 
       defineResolvers(schema, typeToServiceMap, serviceMap, typeFieldsToService)
 
-      return schema
+      return { schema, failedMandatoryServices }
```

Comment:
I'm guessing this will be a semver major change? We should update the typings (with type tests) as well: https://github.com/mercurius-js/mercurius/blob/6943453acd1f0bc1787d1b23695230227e890c58/index.d.ts#L230

Reply:
It's better to avoid the breaking change.